packages/notte-agent/src/notte_agent/falco/agent.py

```
@@ -3,10 +3,14 @@
 import typing
 from collections.abc import Callable
 from enum import StrEnum
+from typing import Union
```

Ruff (0.8.2)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)